client/src/components/FundsMenu.tsx

```diff
@@ -38,6 +38,7 @@ export interface FundsMenuState {
   editMode: boolean;
   newFundName: string;
   dialog?: JSX.Element;
+  adding: boolean;
```

Comment:
I'm not sure as to what the purpose of this is. Is it to say if a user is in the process of adding a company and/or fund?